### ISSUE TITLE
Update not possible - Not enough Space

### ISSUE THREAD

**Issue by paparansen** [rg35xx-cfw/Koriki#92]
### What happened?

Update to 1.0.2 is not possible, because there is not enough space
to copy the files from the ZIP to first Partition.
Tried with fresh & 1 time started 1.0 & 1.0.1
First Partition is 2.99GB, shouldnt this be 5.99GB ?

**Comment by acmeplus** [comment]
You have to delete the existing files before. Otherwise it does not have enough temporary space to write the new files.

**Comment by paparansen** [comment]
ALL files?

**Comment by acmeplus** [comment]
Yeah, but technically you can just delete boot/batocera to free space and then copy the files over

**Comment by paparansen** [comment]
thx, that worked... but now there is different & ugly  boot logo ^^
also, shutdown from menu not working - it just reloads emulation station.
and no kodi :( any way to add kodi ?

**Comment by benni092** [comment]
Jeah had the same issue. I dont know what file contains the bootscreen, but just copy all other files from a previous release.

**Comment by acmeplus** [comment]
I was tired of the original logo. The logo is boot_logo.bmp.gz I believe. You can download the boot.tar.xz from v1.0.1 and copy that logo over

**Comment by paparansen** [comment]
found a some other bugs/issues.
should i open 1 thread for all, or 1 thread for each ?

**Comment by acmeplus** [comment]
> found a some other bugs/issues. should i open 1 thread for all, or 1 thread for each ?

Add those here first

**Comment by paparansen** [comment]
-- shutdown from menu not working - it just reloads emulation station
-- volume at max after power on - previous setting not recognized i guess
-- brightness at max after power on - previous setting not recognized i guess
-- when using hotkey & vol for brightness - volume changes too
-- after slepp/wakeup - controller mapping comes up
